CMakeLists.txt

```diff
@@ -85,6 +85,26 @@ endif(WITH_CCACHE)
 
 option(WITH_SCCACHE "Build with sccache.")
 if(WITH_SCCACHE)
+  if(NOT FORCE_MAX_JOBS)
+    execute_process(
+      COMMAND "sccache" "--dist-status"
+      OUTPUT_VARIABLE sccache_dist_status
+      RESULT_VARIABLE sccache_dist_status_result
+      COMMAND_ERROR_IS_FATAL ANY
```

Comment:
again, this was introduced in CMake 3.19, might want to check for the cmake version before using it, or bump the minimal required version of CMake. and the same suggestion applies in another comment in this PR, regarding bumping up the CMake version.

Reply:
I dropped that line; the command still exits with `0` if distributed compilation is disabled, so instead we'll look for "disabled" in its output.